examples/README.md

```diff
@@ -20,6 +20,8 @@ Training on WATTS is enabled with a suite of simple examples:
 * [__Optimization_SAM-OpenMC_VHTR-scipy__](Optimization_SAM-OpenMC_VHTR-scipy/README.md) - 1-criteria optimization of SAM/OpenMC workflow using scipy
 * [__Optimization_SAM-OpenMC_VHTR-pymoo__](Optimization_SAM-OpenMC_VHTR-pymoo/README.md) - Multi-criteria optimization of SAM/OpenMC workflow using pymoo
 ## Examples with Sensitivity studies
+* [__Optimization_PyARC_DAKOTA__](Optimization_PyARC_DAKOTA/README.md) - Multi-criteria optimization of PyArc workflow using Dakota
```

Comment:
please update to `PyARC` - not `PyArc` - this change is needed at other places.

Reply:
The typos have been corrected.